fixtures/questionparser.py

```diff
@@ -0,0 +1,43 @@
+import os, re
+
+from codingfix import fix_coding
+
+
+def parse_file(path):
+    question_pattern = r'P\n' \
+                       '%(?P<subject>.*?)%\n' \
+                       '(\$(?P<image>([^\$]|\n)*?)\$\n)?' \
+                       '@(?P<question>([^@]|\n)*?)@\n' \
+                       '@(?P<answers>([^@]|\n)*?)@\s*?'
+
+    questions = list()
+    with open(path, 'r') as f:
+        matches = re.finditer(re.compile(question_pattern), f.read())
+        for match in matches:
+            group_names = ('subject', 'question', 'answers', 'image')
+            question = {group_name: match.group(group_name) for group_name in group_names if match.group(group_name)}
+            question = fix_coding(question)
+            questions.append(question)
+            assert 'answers' in question and 'subject' in question and 'question' in question
```

Comment:
If the assert doesn't work that means that the questions do not hold to the requested format e.g. there were no answers specified. There shouldn't be any issues with the latest version of questions database, I tested it and it runs smoothly